sdk/storage/azure-storage-blob/azure/storage/blob/aio/_container_client_async.py

```diff
@@ -1379,7 +1376,7 @@ async def delete_blobs(
                 :caption: Deleting multiple blobs.
         """
         if len(blobs) == 0:
-            return iter([])
+            return iter([])  # type: ignore [return-value]
```

Comment:
Hmm this is technically incorrect. We are returning a synchronous iterator when we say we return an async one. This would break someone using `async for`. I'm not quite sure how to return an empty AsyncIterator though. There is the `aiter` function but that was added in Python 3.10 so we can just use that.

Reply:
So just leave this ignore in for now? (since our min Python dep is only 3.7 sadly)

Reply:
I think we should try and fix it because its a GitHub issue waiting to happen. Let's ask Krista/Anna an easy way to return an empty AsyncIterable.

Reply:
This aged well 😄  🥛